sw/device/tests/otbn_smoketest.c

```diff
@@ -130,6 +133,10 @@ static void test_barrett384(otbn_t *otbn_ctx) {
 static void test_err_test(otbn_t *otbn_ctx) {
   CHECK(otbn_load_app(otbn_ctx, kAppErrTest) == kOtbnOk);
 
+  // TODO: Turn on software_errs_fatal for err_test. Currently the model doesn't
+  // support this feature so turning it on leads to a failure when run with the
+  // model.
```

Comment:
Probably the thing to do is to have a `set_ext_reg` method on the stepped interface and mirror (a tiny bit of) the TL decoding logic in `otbn_core_model`. This would also let us get rid of our current hacky `start` support, since it would be subsumed by the more general stuff. Probably raise an issue: I'd guess it's a day or two's work.